mockagent_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package jaeger
+
+import (
+	"context"
+	"net"
+	"sync"
+	"testing"
+
+	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/monkit-jaeger/gen-go/agent"
+	"storj.io/monkit-jaeger/gen-go/jaeger"
+)
+
+// StartMockAgent starts a mock agent on a local udp port.
+func StartMockAgent(t *testing.T, f func(mock *MockAgent)) {
```

Comment:
This looks good, but I would rename this function to something more specific. Something like `withMockAgent` or `runWithAgent`.